src/node_modules/ls-markup/eval_defs.js

```diff
@@ -270,5 +270,14 @@ module.exports.EXPRESSIONS = {
 				``
 			];
 		}
-	}
+	},
+	iframe: {
+        has: { 'url': ARGTYPES.str },
+        handle: (node) => {
+            return [
+                `<iframe src=${node.data.url} width="100%" height="100%" frameborder="0"/>`,
+                ``
```

Comment:
I believe iframes aren't self-closing so you should return `</iframe>` in the second item of the array instead of using `<iframe ... />` in the first one. I think it'd also make sense to expose the `width` and `height` parameters to users instead of hardcoding them to 100%. Otherwise the changes look good.

Reply:
Done! But just a question... is it possible to have a default value (100% in this case) for the attributes?

Reply:
Great! Currently it's not possible to set default values for parameters. I'll give this a spin soon and merge it if everything works as expected.